sig-auth/governance.md

```diff
@@ -0,0 +1,153 @@
+  - Establish new subprojects
+  - Decommission existing subprojects
+  - Seed technical leads nominated by legacy SIG leads from subproject owners
+  - *MUST* ensure that all areas of the SIG (including those that fall outside subprojects) have
+    active owners.
+  - *MAY* set SIG milestone priorities.
+  - *SHOULD* act as an escalation point for technical disputes in subprojects.
+  - *MUST* resolve issues impacting multiple subprojects in the SIG.
+  - *SHOULD* meet or communicate regularly enough to ensure they are aligned. Decisions made by
+    technical leads should be consistent across the SIG. If technical leads are providing
+    inconsistent direction, then they *MUST* align themselves.
+  - Technical Leads *MAY* select additional tech leads through a [super-majority] vote amongst tech
+    leads and chairs.  This *SHOULD* be supported by a majority of SIG Members under
+    [lazy-consensus].
+  - Technical Leads *MUST* remain active in the role and are automatically removed from the position
+    if they are unresponsive for > 3 months and *MAY* be removed if not proactively working with
+    other technical leads to fulfill responsibilities. Technical leads *MAY* arrange a short-term
+    (<6 months) leave of absence with coordinated coverage of responsibilities.
+  - Technical Leads *SHOULD* represent a diverse set of organizations.  This is predicated on
+    interest in the role from multiple organizations with qualified candidates.
+  - *SHOULD* have a record of consistent solid technical judgement and leadership over a sustained
+    period especially over areas for which they are the de facto lead
+  - Number: 3-7
+  - Defined in [sigs.yaml] and [OWNERS] files
+
+
+
+- **Subproject Owners**
+  - Scoped to a subproject defined in [sigs.yaml]
+  - Seed subproject owners *MAY* include initial proposal author(s) and/or Tech Leads
+  - *MUST* be an escalation point for technical decisions and failing or flaky tests in the
+    subproject
+  - *MUST* proactively maintain the subproject health and status or delegate this
+    responsibility. For actively developed projects, this *SHOULD* include setting milestone
+    priorities, tracking release bits, and ensure adequate test coverage. For maintenance mode
+    projects, this *SHOULD* include maintaining test health and ensuring compatibility with new
+    features.
+  - *MUST* remain active in the role and are automatically removed from the position if they are
+    unresponsive for > 3 months. Subproject owners *MAY* arrange a short-term (<6 months) leave of
+    absence with coordinated coverage of responsibilities.
+  - *SHOULD* have a record of consistent solid technical judgement over a sustained period,
+    especially over areas for which they are the owner
+  - *MAY* be removed if not proactively working with other Subproject Owners to fulfill
+    responsibilities.
+  - *MAY* decide to step down at any time and propose a replacement.  Use [lazy-consensus] amongst
```

Comment:
If a subproject has no effective leadership would it fall to majority of the SIG or the TLs of the SIG to solve/address this?